### ISSUE TITLE
Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by Khyati007** [khyatit13/Coding-practice#219]
Source: `<span class="c-coordinateSummaryPanel__summaryStatsNum c-coordinateSummaryPanel__summaryStatsNum--blue t-jobSummaryTotal2">11</span>`
Element Location: .t-jobSummaryTotal2
failureSummary: Fix any of the following:
  Element has insufficient color contrast of 4.14 (foreground color: #007ac2, background color: #f3f3f3, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1Tags: [Ljava.lang.String;@4264b240
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
